### DISCUSSION TITLE
No available nodes

### DISCUSSION THREAD

**Discussion by QuirkyCarlos** [lavalink-devs/Lavalink#1066] [Q&A] [answered]
Hello ive looked through the FAQ and Common issues and couldnt find this I self host the Lavalink in a Docker Thats online and ready for connections but my bot says like No Available nodes and not sure what it means XD i know i sould like a complete noob at this but anyhelp would mean alot 

Many Thanks
QuirkyCarlos

Scrrenshot 
https://i.imgur.com/MxClHvY.png

**Comment by topi314** [comment, marked as answer]
that prob means you didnt connect a node. thats something you should ask your client devs on how to fix
